src/configuration/comment-types.ts

```diff
@@ -24,15 +27,11 @@ export enum CommentType {
    */
   CONTRIBUTOR = 0b100000,
   /**
-   * A user comment action on a {@link CommentType.ISSUE} or {@link CommentType.REVIEW}
-   */
-  COMMENTED = 0b1000000,
-  /**
-   * Pull request opening item
-   */
-  TASK = 0b10000000,
```

Comment:
`COMMENTED` is not needed anymore because it is implied. `TASK` is now `SPECIFICATION` (used to be `ISSUE_SPECIFICATION` and `PULL_TASK`, not it is `PULL_SPECIFICATION`).